scripts/build/build.ps1

```diff
@@ -12,7 +12,7 @@ $ErrorActionPreference="Stop"
 
 . "$PSScriptRoot\..\common\_common.ps1"
 
-_ "$RepoRoot\scripts\build\generate-version.ps1"
+. "$RepoRoot\scripts\build\generate-version.ps1"
```

Comment:
@brthor the "_" is equivalent to "call" (in this repo) where "." is equivalent to include. generate-version does set bunch of variables which get lost when you call the script and return, including it preserves them